test/Middleware/WhoopsErrorResponseGeneratorTest.php

```diff
@@ -142,8 +142,8 @@ public function testJsonContentTypeResponseWithJsonResponseHandler()
         $this->request->getParsedBody()->willReturn([]);
 
         $this->response->withHeader('Content-Type', 'application/json')->will([$this->response, 'reveal']);
-        $this->response->withStatus(StatusCode::STATUS_INTERNAL_SERVER_ERROR)->will([$this->response, 'reveal']);
-        $this->response->getStatusCode()->willReturn(StatusCode::STATUS_INTERNAL_SERVER_ERROR);
+        $this->response->withStatus(StatusCode::STATUS_IM_A_TEAPOT)->will([$this->response, 'reveal']);
+        $this->response->getStatusCode()->willReturn(StatusCode::STATUS_IM_A_TEAPOT);
```

Comment:
Not sure I understand this change. `$error` here is a simple `RuntimeException`, so I'd expect an internal server error. Or are you redefining the `$error` somehow?